src/python/WMCore/Storage/Backends/XRDCPImpl.py

```diff
@@ -111,6 +112,9 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
                     for initFile in initFiles:
                         copyCommand += "source %s\n" % initFile
 
+        if args.disablewriterecovery and not args.cerncastor:
+            copyCommand += "env XRD_WRITERECOVERY=0 "
```

Comment:
no, why would there ? env is a command pre-processort, it changes the environment for the command it executes.